x-pack/solutions/security/test/serverless/functional/test_suites/ftr/cloud_security_posture/mki_only/agentless/mki_create_agent.ts

```diff
@@ -44,10 +44,27 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       await cisIntegration.inputIntegrationName(integrationPolicyName);
 
       await cisIntegration.selectSetupTechnology('agentless');
-      await cisIntegration.selectAwsCredentials('direct');
+
+      try {
```

Comment:
Stabilizing UI test code for situations like this can be tricky as we usually don't have good indicators to wait for when interacting with certain components.  A pattern that's sometimes used to deal with that sort of flakiness is self-checking the selected value and retry if selection failed or if it didn't select the right thing, e.g. something like (pseudo code)
```
retry for time x {
    select value
    get selected value
    if selected value is not the expected value: throw error
}
```
Did you consider that approach?